html/global_attributes.json

```diff
@@ -883,10 +883,24 @@
               "version_added": false
```

Comment:
```suggestion
              "version_added": "79"
```

Can you also update the rest of this data to match the IDL attribute after applying those suggestions?